blueprint/index.md

```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
 
-3. Select the *Send-SMS.custom.json* file and associate with the web services data action you created in the [Add a web services data actions integration](#add-a-web-services-data-actions-integration "Goes to the Add a web services data actions integration section") section and click **Import Action**.
+3. Select the `Send-SMS.custom.json` file and associate with the web services data action you created in the [Add a web services data actions integration](#add-a-web-services-data-actions-integration "Goes to the Add a web services data actions integration section") section and click **Import Action**.
 
    ![Import the Send the SMS data action](images/5BImportSendSMSDataAction.png "Import the SMS data action")
 
 ### Import and publish the script
-You need to import the script *Send-SMS-with-Teams-Video-URL.script* that references the created data actions. The script generates the **Escalate to Teams** button for agents during an active interaction with the customer. It also sends an SMS that contains the Microsoft Teams video URL to the customer.
+Import the `Send-SMS-with-Teams-Video-URL.script` file that references the created data actions. The script creates the **Escalate to Teams** button for agents during an active interaction with the customer. When the agent clicks the button, it sends an SMS that contains the Microsoft Teams video URL to the customer.
 
-1. Download the *Send-SMS-with-Teams-Video-URL.script* file from the [microsoft-teams-blueprint repo](https://github.com/MyPureCloud/microsoft-teams-blueprint "Opens the GitHub repo") GitHub repository. Save this file to your local desktop to import it into Genesys Cloud.  
+1. Download the `Send-SMS-with-Teams-Video-URL.script` file from the [microsoft-teams-blueprint repo](https://github.com/MyPureCloud/microsoft-teams-blueprint "Opens the GitHub repo") GitHub repository. Save this file to your local desktop to import it into Genesys Cloud.  
 
 2. Navigate to **Admin** > **Contact Center** > **Scripts** and click **Import**.
 
    ![Import the script](images/6AImportScript.png "Import the script")
 
-3. Select the downloaded *Send-SMS-with-Teams-Video-URL.script* file.
+3. Select the downloaded `Send-SMS-with-Teams-Video-URL.script` file.
 
    ![Import the send SMS script](images/6BImportSendSMSScript.png "Import the SMS script")
 
 4. To configure the script for use in an outbound message, open the imported script.
 
    ![Open the Script menu](images/6COpenScriptDropdown.png "Open the Script menu")
 
-5. Click the **Actions** icon and then click **Escalate to Teams**
+5. Click the **Actions** icon and then select **Escalate to Teams**.
 
-  ![Click Script Actions](images/clickScriptActions.png "Click Script Actions")
+   ![Click Script Actions](images/clickScriptActions.png "Click Script Actions")
 
-6. Expand the First Data Action
+6. Expand the first data action **Data Actions.Execute Data Action**, that creates the Teams meeting.
```

Comment:
I found this a little confusing. How about, "Expand the first data action called **Data Actions. Execute Data Action**. This data action creates the Teams meeting.